### PR TITLE
Apply freezing fee

### PR DESCRIPTION
**Pull request by MichaelKim20**
Freezing TXs now have to include SlashAmountPenatly amount of fee for each Freeze output they have
It's a related PR in Agora.
https://github.com/bosagora/agora/pull/2597

### CODE REVIEW

File: src/modules/wallet/WalletTxBuilder.ts [hunk 1]
```diff
@@ -2073,7 +2117,19 @@ export class WalletCancelBuilder extends WalletTxBuilder {
         const sum_r = r.reduce<Amount>((prev, value) => Amount.add(prev, value.amount), Amount.make(0));
         const fee = Amount.subtract(sum_s, sum_r);
         const fee_payload = TxPayloadFee.getFeeAmount(this._tx.payload.length);
-        const fee_tx = Amount.subtract(fee, fee_payload);
+        const fee_freezing = tx.isFreeze() ? Constant.SlashPenaltyAmount : Amount.make(0);
+        let fee_tx: Amount;
+        if (Amount.greaterThanOrEqual(fee, fee_payload)) {
```
Comment (TrustHenry) [inline src/modules/wallet/WalletTxBuilder.ts:2122]: If the transaction type is payment,
fee = 0.01 & feepayload = 0.01 
If so, the fee_tx will be zero.

Comment (MichaelKim20) [reply src/modules/wallet/WalletTxBuilder.ts:2122]: `fee` = `sum of input` - `sum of output`;
`fee` = `fee_tx` + `fee_payload` + `fee_freezing`;
if fee_tx is 0.01, fee_payload is 0.2 then
fee = 0.01 + 0.2 = 0.21

Comment (MichaelKim20) [reply src/modules/wallet/WalletTxBuilder.ts:2122]: 이전에 생성된 트랜잭션이 정상적으로 만들어 졌다는 가정이므로 위의 공식으로 계산합니다.

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**